recipes/cwltest/meta.yaml

```diff
@@ -1,31 +1,29 @@
-{% set name = "cwltest" %}
 {% set version = "2.0.20200220223835" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: cwltest
+  version: {{ version }}
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```

Comment:
Your change here seems to be in the opposite direction of what I’ve seen in other recipes. Is there a policy somewhere on how PyPI URLs should be written in recipes?

(I agree with your change and find it more readable, but I’m worried others would “correct” this by re-introducing the `{{ name }}` variable.)

Reply:
No, I don't think we have a policy.
The overuse of Jinja templating is just what `conda-skeleton` outputs, hence it is quite common.
Johannes started some work to counteract that bloat in https://github.com/conda/conda-build/pull/3176.
I'm actually not sure where most of the other folks stand on this. At least there was a comment at https://github.com/conda/conda-build/pull/3176#issuecomment-427454913 saying others in conda-forge gradually reduced the amount of template use.
It's not too unlikely someone (Johannes, myself, or maybe Marcelo Trevisani) will work on this some time this year, I guess.

Reply:
Thanks, this was some interesting reading. Then I’m inclined to follow the simpler style myself and I’ll also suggest it when reviewing recipes.